External/FEXCore/include/FEXCore/Core/CoreState.h

```diff
@@ -144,6 +144,37 @@ namespace FEXCore::Core {
       uint64_t L1Pointer{};
       /**  @} */
     } X86;
+
+    struct {
+      // Process specific
+      uint64_t PrintValue{};
+      uint64_t PrintVectorValue{};
+      uint64_t RemoveCodeEntryFromJIT{};
+      uint64_t CPUIDObj{};
+      uint64_t CPUIDFunction{};
+      uint64_t SyscallHandlerObj{};
+      uint64_t SyscallHandlerFunc{};
+
+      uint64_t FallbackHandlerPointers[FallbackHandlerIndex::OPINDEX_MAX];
+
+      // Thread Specific
+      uint64_t SignalHandlerRefCountPointer{};
+
+      /**
+       * @name Dispatcher pointers
+       * @{ */
+      uint64_t DispatcherLoopTop{};
+      uint64_t DispatcherLoopTopFillSRA{};
+      uint64_t ThreadStopHandlerSpillSRA{};
+      uint64_t ThreadPauseHandlerSpillSRA{};
+      uint64_t ThreadStopHandler{};
+      uint64_t ThreadPauseHandler{};
+      uint64_t UnimplementedInstructionHandler{};
+      uint64_t OverflowExceptionHandler{};
+      uint64_t SignalReturnHandler{};
+      uint64_t L1Pointer{};
+      /**  @} */
+    } RISCV;
```

Comment:
Should this be wrapped in an `#ifdef`?

Reply:
We do support building non-current-host backends, though not sure if that would work with the RISCV backend.